--- a/usr/local/share/ueye/bin/ueyesdk-setup.sh
+++ b/usr/local/share/ueye/bin/ueyesdk-setup.sh
@@ -1,4 +1,7 @@
 #!/bin/sh
+
+if [ -f /usr/local/share/ueye/installed.txt ]; then exit; fi
+
 #Auto-generated embedded sdk setup script, generated Fr 20. Okt 10:31:57 CEST 2017.
 chown root:root "/usr/local/share/ueye/ueyeusbd/ueyeusbd"
 chown root:root "/usr/local/share/ueye/bin/ueyenotify"
@@ -96,5 +99,7 @@
 else
     echo "Warning: Failed to run ldconfig. Please run ldconfig manually or reboot your system."
 fi
+
+echo "uEye SDK installed" > /usr/local/share/ueye/installed.txt
 
 #End of auto-generated embedded sdk setup script.
